spring-cloud-gcp-kotlin-samples/spring-cloud-gcp-kotlin-app-sample/README.adoc

```diff
@@ -0,0 +1,48 @@
+= Spring Cloud GCP Kotlin Sample App
+
+This is a simple Kotlin Spring Boot application which demonstrates how to integrate with Spring Cloud GCP libraries from Kotlin.
+It sets up a registration service in which a someone publishes registrant names to a Cloud Pub/Sub topic; these names are then persisted to a Cloud SQL database.
+
+This application demonstrates the following Spring Cloud GCP integrations:
```

Comment:
You should also mention which parts of Spring it demonstrates, namely, Spring Integrations and Spring Data JPA.